pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin.go

```diff
@@ -65,6 +66,12 @@ func (p plugin) Apply(rs *core_xds.ResourceSet, ctx xds_context.Context, proxy *
 			return err
 		}
 	}
+
+	err := applyToRealResources(rs, policies.ToRules.ResourceRules, ctx.Mesh)
```

Comment:
Does support of existing policies (kind: MeshService and name == kuma.io/service) is a part of this task?(Old policies apply to new listeners)

Reply:
No, this is only to add support for real MeshService. 

Also when you have old policy targeting `MeshService` and there is real MeshService with the same name this will work. I think this will work for universal, but on k8s we have a different naming scheme. 

For this to work on k8s, we would need to update how we build ResourceRules to seek for real MeshService based on `kuma.io/service` tag

Reply:
Ok, it seems like after MADR related to migration we can add this